app/DB_Connectors/MYSQL.py

```diff
@@ -0,0 +1,68 @@
+import pandas
+import pymysql
+
+from creds.CRED_holder import Cred
+from logs.Logger.Loger import DataLogger
+
+
+class MYSQLConnector:
+
+    def __init__(self):
+        self.info = Cred('mysql')
+        self._logger = DataLogger(self)
+
+    def __connect__(self):

```

Comment:
```suggestion
    def __connect(self):

```

not a magic method, no need to attach the closing underscores